test/roundtrip.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ *
+ * Testing package conversion from frictionless to CKAN
+ * The tests have been reused from original library
+ * https://github.com/frictionlessdata/frictionless-ckan-mapper
+ *
+ */
+
+const test = require('ava')
+const fs = require('fs')
+
+const {
+  packageCkanToFrictionless,
```

Comment:
I think we could have slightly nicer naming but i would have to see the lib itself to be sure i.e. it would be nice for this to be similar to python (maybe functional naming e.g.)

```
ckanToFrictionless.dataset
ckanToFrictionless.resource
```
Or if we have full naming ...

```
ckanToFrictionlessDataset
ckanToFrictionlessResource
```

Or even

```
c2fDataset
c2fResource

f2cDataset
f2cResource
```

Reply:
I like that last option, will do that!